src/wp-includes/class-wp-comment-query.php

```diff
@@ -992,6 +988,12 @@ private function set_found_comments() {
 		global $wpdb;
 
 		if ( $this->query_vars['number'] && ! $this->query_vars['no_found_rows'] ) {
+			$found_request = "SELECT COUNT(*)
+			{$this->sql_clauses['from']}
+			{$this->sql_clauses['where']}
+			{$this->sql_clauses['groupby']}
+			{$this->sql_clauses['orderby']}
```

Comment:
For count value from the quer,y i don't think we need `groupby` and `orderby`. Is there any use case in which we need these clauses?